packages/console/src/pages/EnterpriseSsoDetails/Connection/OidcMetadataForm/index.tsx

```diff
@@ -39,13 +40,18 @@ function OidcMetadataForm({ providerConfig, config, providerName }: Props) {
         </InlineNotification>
       )}
       <FormField isRequired title="enterprise_sso.metadata.oidc.client_id_field_name">
-        <TextInput {...register('clientId', { required: true })} error={Boolean(errors.clientId)} />
+        <TextInput
+          {...register('clientId', { required: true })}
+          error={Boolean(errors.clientId)}
+          placeholder="Client ID"
```

Comment:
nit: Should use i18n for placeholders

Reply:
Can update this later, should align all forms.